packages/dapp-svelte-wallet/ui/package.json

```diff
@@ -12,13 +12,14 @@
   },
   "devDependencies": {
     "@agoric/captp": "^1.3.3",
-    "@agoric/harden": "^0.0.6",
     "@rollup/plugin-commonjs": "^12.0.0",
     "@rollup/plugin-node-resolve": "^8.0.0",
+    "livereload-js": "file:livereload-js-v3.3.1.tgz",
     "rollup": "^2.3.4",
     "rollup-plugin-livereload": "^1.0.0",
     "rollup-plugin-svelte": "^5.0.3",
     "rollup-plugin-terser": "^6.1.0",
+    "ses": "^0.10.0",
```

Comment:
is it correct for this to be a devDependency?

Reply:
I suspect not, most packages import `install-ses` if anything. I'll take a closer look.

Reply:
ah, ok, this package defines it's own version of `@agoric/install-ses` (with more taming than `@agoric/install-ses` uses), so yes it needs `ses` as a direct dependency

Reply:
also it uses the new `ses/lockdown` subset, which is less than what `@agoric/install-ses` provides (`Compartment`)

Reply:
It's correct.  We import `ses/lockdown` directly to avoid taking a babel dependency.